dev-portal/src/pages/Admin/Admin.jsx

```diff
@@ -4,17 +4,21 @@ import { BrowserRouter as Router } from 'react-router-dom'
 import { ApiManagement, SideNav } from './'
 import { AdminRoute } from './../../';
 
+import PageWithSidebar from 'components/PageWithSidebar'
+
 export class Admin extends Component {
   render() {
     return (
       <Router>
-        <div style={{ display: "flex", flex: "1 1 auto", overflow: "hidden" }}>
-          <SideNav style={{ flex: "0 0 auto" }} />
-          <div style={{ flex: "1 1 auto", overflow: 'auto' }}>
-            <AdminRoute exact path="/admin" component={ApiManagement} />
-            <AdminRoute path="/admin/apis" component={ApiManagement} />
-          </div>
-        </div>
+        <PageWithSidebar
+          sidebarContent={<SideNav />}
+          content={
+            <React.Fragment>
+              <AdminRoute exact path="/admin" component={ApiManagement} />
+              <AdminRoute path="/admin/apis" component={ApiManagement} />
+            </React.Fragment>
+          }
+        />
```

Comment:
Super cool! Can you change PageWithSidebar to accept children instead of content?